contracts/test/mocks/KernelDepositableMock.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.4.24;
```

Comment:
I'm not sure that we really need this test, as we are testing something that our codebase doesn't offer, right? Besides I don't think I like the idea of a depositable kernel, but maybe it's because I'm missing some possible use cases.

Reply:
I'd like to keep it, as it's functionality that _could_ be enabled, even if it's not directly offered.